recipes/nbgrader/build.sh

```diff
@@ -0,0 +1,4 @@
+"${PYTHON}" setup.py install --single-version-externally-managed --record=record.txt
+
+#"${PREFIX}/bin/jupyter-nbextension" install --sys-prefix --overwrite --py nbgrader
```

Comment:
At the moment it is unnecessary :) It is a place holder for the `0.4.x nbgrader` feature release.